### ISSUE TITLE
Add a Events Page in the Website which Shows all the old events and all the Upcoming events.

### ISSUE THREAD

**Issue by Mr-Sunglasses** [FOSS-Community/website-fossc#186]
Create a Page in the website which can show these following details of all the events, We can also make an API which stores the information of all the Events.

**Comment by Utkarsh4517** [comment]
OFF TOPIC TO website-fossc.

Can I add this feature in fossc-app? It will show past events and list of upcoming events or talks with regirstration links as a visitor and as well as a Volunteer.

**Comment by irohanrajput** [comment]
Can I work on this ?

**Comment by praptisharma28** [comment]
Assign me @Mr-Sunglasses

**Comment by Mr-Sunglasses** [comment]
Cc. @codecShivam

**Comment by codecShivam** [comment]
Go ahead! @praptisharma28

**Comment by Mr-Sunglasses** [comment]
> @praptisharma28 you working on it ?

Ig. @praptisharma28 is busy these days, let give this issue to someone else.

**Comment by codecShivam** [comment]
> > @praptisharma28 you working on it ?
> 
> Ig. @praptisharma28 is busy these days, let give this issue to someone else.

Yeah! Let's do this work for the new website that's going to be even better!
